tests/integration-tests/tests/update/test_update.py

```diff
@@ -1529,6 +1533,113 @@ def test_dynamic_file_systems_update_rollback(
     )
 
 
+@pytest.mark.usefixtures("instance")
+def test_dynamic_file_systems_update_data_loss(
```

Comment:
Instead of a new integ test, can this logic be added to an existing one?

Reply:
This was my initial thought. I changed my mind for the following reasons:
1. **time:** the main DFSM test is already long (~2h); including this test would increase that time by ~15min. We recently invested effort in reducing the main DFSM integ test; including data loss test would go in the opposite direction
2. **complexity:** DFSM integ test is already complicated: iterating on new logic is frustrating because you need to comments out parts here and there to troubleshoot issues.

What do you think?

Reply:
There seem to be several competing priorities for integration tests and we don't have clear direction on what the right path is.  I think it makes sense to have separate tests since they will likely be run in parallel and eventually I think we want to use shared resources for these tests so they eventually consume less time overall.  But there was also a push to consolidate tests to have fewer of them, so it's not clear where the line is.  I think I will disagree and commit on this one and we need to take a long hard look at integ tests again to decide what our priorities really are.